docs/resources/cdn_domain.md

```diff
@@ -36,6 +36,9 @@ The following arguments are supported:
 * `sources` - (Required, List, ForceNew) An array of one or more objects specifies the domain name of the origin server.
   The sources object structure is documented below.
 
+* `service_area` - (Optional, String, ForceNew) The area covered by the acceleration service. Valid values are
```

Comment:
I saw on the console that `service_area`  is a required field.

Reply:
it defaults to mainland_china..., make it required will break current customers configuration.
so it works before without this parameter :P

Reply:
I see.